VideoApp/VideoApp/Video/Tracks/Camera/CameraConfigFactory.swift

```diff
@@ -36,13 +36,17 @@ class CameraConfigFactory {
             
             cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
             frameRate = 20
-        case .vp8Simulcast:
+        case .vp8SimulcastVGA:
+            targetSize = CMVideoDimensions(width: 640, height: 480)
+            cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
+            frameRate = 24 // With VGA simulcast enabled there are 2 temporal layers, allowing a frame rate of f/4
```

Comment:
Is this comment I made accurate?

Also I don't really understand the existing comment below referring to 1024 x 768 because the code is using 900 x 720?

Reply:
There are still 3 temporal layers even when simulcasting with VGA resolutions.

```suggestion
            frameRate = 24 // With VGA simulcast enabled there are 3 temporal layers, allowing a frame rate of {f, f/2, f/4}
```